package.json

```diff
@@ -56,6 +56,7 @@
 	},
 	"scripts": {
 		"api-docs": "jsdoc --configure .jsdoc.json",
+		"imports-gen": "node ./build/imports-generator --axios 'axios' --doT 'dot' --CssSelectorParser 'css-selector-parser'",
```

Comment:
I'm concerned that if we keep this mapping as CLI flags, the list may become very difficult to maintain in the future.

Why not just add the mapping to `./build/imports-generator.js`?

```js
const mapping = [ ['axios','axios'], ['doT', 'dot'], ['CssSelectorParser', 'css-selector-parser'] ]
```

Reply:
On second thought moved it to mapping...